src/admin/vote/Add.tsx

```
@@ -0,0 +1,356 @@
+              )}
+            </div>
+
+            {showSuggestions && (
+              <mdui-tabs
+                value={classes.sort((a, b) => a.grade - b.grade)[0].id}
+              >
+                {classes
+                  .sort((a, b) => a.grade - b.grade)
+                  .map((c) => (
+                    <mdui-tab value={c.id}>
+                      Klasse {c.grade}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)